package/usr/local/bin/hassbian-config

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
-
 SUITE_INSTALL_DIR=/opt/hassbian/suites
 SUITE_INSTALL_DB=/srv/homeassistant/hassbian/suite-states
 SUITE_INSTALL_DB_LOCK=/srv/homeassistant/hassbian/suite-states.lock
+SUITE_CONTROL_DIR=/opt/hassbian/control
 
 function usage {
    echo $0: usage:
    echo
    echo $0 \<command\> \<suite\>
    echo where \<command\> is one of:
-   echo     install  - installs a software suite
-   echo     upgrade  - upgrades a software suite
-   echo     show     - shows software suites available
+   echo     install        - installs a software suite
+   echo     upgrade        - upgrades a software suite
+   echo     show           - shows software suites available
+   echo     show-installed - list all packages installed with `hassbian-config`
```

Comment:
Would it be possible to mark the installed ones in the show menu instead? Seems redundant to do it like this?

Reply:
Yes, definitely!

Reply:
![image](https://user-images.githubusercontent.com/15093472/34845069-f8ca764a-f713-11e7-8f73-3c246362200b.png)
Prints "installed" if installed after this change is implemented.
But it want to keep the "show-installed" function (removed from usage overview), this will be nice before reinstalling.